src/scripts/main.js

```diff
@@ -1 +1,108 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+function handleFirstClick() {
+  firstPromise();
```

Comment:
The same issue here: `firstPromise()` is called without being defined. It is recommended to define this function before using it.